src/validation/error.messages.ts

```diff
@@ -58,5 +58,15 @@ export enum ErrorMessages {
   MAX_OCCUPATION_LENGTH = "Occupation must be 100 characters or less",
   MAX_ROLE_LENGTH = "Role and responsibilities must be 4000 characters or less",
   // Invalid characters
-  INVALID_CHARACTERS_FULL_NAME = "Full name must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes"
+  INVALID_CHARACTERS = " must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
+  FULL_NAME_PREFIX = "Full name",
```

Comment:
Rather than do the string constant validation in all the .validation files I'd suggest defining the strings (using concatenation) here and just referencing them. E.g.

FIRST_NAME_INVALID_CHARACTERS = "First name" + INVALID_CHARACTERS,

Reply:
That's a shame. Should we consider changing this to just a list of string constants then (where presumably string concatenation would be allowed)? It's just that FULL_NAME_PREFIX on its own isn't really an error message and neither is INVALID_CHARACTERS. Or, are we happy that the `ErrorMessages` enum is just a bucket of strings for error messages which can define the entire message or sometimes need to be concatenated together.

Reply:
I had similar thoughts but I'd prefer to raise a ticket and link to this one as it seems like a wider issue to have come out of this work.

Reply:
I think to keep to follow the enum standard, you could have the prefix and INVALID_CHARACTERS in the same file but moved outside of the enum (and don't export them), then in the enum, you could add extra error message types by concatenating the strings like
INVALID_CHARACTERS_FIRST_NAME = FIRST_NAME_PREFIX + INVALID_CHARACTERS.
That way INVALID_CHARACTERS_FIRST_NAME would still be a'type' of errorMessage